cluster/job_once_example_test.go

```diff
@@ -0,0 +1,40 @@
+package cluster
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-server/v5/plugin"
+)
+
+func HandleJobOnceCalls(key string) {
+	if key == "the key i'm watching for" {
+		// Work to do only once per cluster
+	}
+}
+
+func ExampleScheduleOnce() {
+	// Use p.API from your plugin instead.
+	pluginAPI := plugin.API(nil)
+
+	// Get the scheduler, which you can pass throughout the plugin...
+	scheduler := GetJobOnceScheduler(pluginAPI)
+
+	// Set the plugin's callback handler
+	_ = scheduler.SetCallback(HandleJobOnceCalls)
```

Comment:
I had that for awhile (one method to set, start and return the scheduler), but then I had to do contortions to use it in the Incident Management -- I need to be able to pass the scheduler to a service (in this case the Incident Service) but I need the service running before I can set the callback (because the callback is a method on the service), and the callback needs to be ready to run when the scheduler starts (because the scheduler will call the callback if there are any scheduled jobs overdue). So I settled on splitting it into these three.